NeuroID/src/main/java/com/neuroid/tracker/events/TouchEventManager.kt

```diff
@@ -101,7 +101,7 @@ abstract class TouchEventManager {
         metadataObj.put("x", motionEvent?.x)
         metadataObj.put("xP", motionEvent?.xPrecision)
         metadataObj.put("xR", motionEvent?.rawX)
-        metadataObj.put("xCalc", motionEvent?.rawX * motionEvent?.xPrecision)
```

Comment:
Looked like the upgrade in the gradle version caused this to expect an Int vs an optional Int. Let me know if I should handle this differently.

Reply:
This is fine.